### PR TITLE
feat: tos-screen

### PR DESCRIPTION
**Pull request by SamTremko**
Tos and privacy policy

### CODE REVIEW

File: apps/mobile/src/components/IconButton.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  style?: StyleProp<ViewStyle>
+}
+
+const PressableStyled = styled(TouchableOpacity)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: ${(p) => p.theme.colors.grey};
+  border-radius: 13px;
+  padding: 7px;
```
Comment (kaladivo) [inline apps/mobile/src/components/IconButton.tsx]: 8px

File: apps/mobile/src/components/Markdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+function Markdown({children}: Props): JSX.Element {
+  const theme = useTheme()
+  return (
+    <MarkdownDisplay
+      style={{
+        strong: css`
+          font-size: 16px;
+          color: #ffffff;
```
Comment (kaladivo) [inline apps/mobile/src/components/Markdown/index.tsx]: ${props => props.theme.colors.white}

File: apps/mobile/src/components/TosFlow/components/TosScreen/components/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  background-color: ${(p) => p.theme.colors.grey};
+  padding: 4px;
+`
+
+const TabButton = styled(Button)<{active: boolean}>`
+  flex: 1;
+  background-color: ${(p) => (p.active ? '#322717' : 'transparent')};
+`
+
+const Gap = styled.View`
```
Comment (kaladivo) [inline apps/mobile/src/components/TosFlow/components/TosScreen/components/Tabs.tsx]: Spacer

File: apps/mobile/src/components/TosFlow/components/TosScreen/images/closeSvg.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import {stringToSvgStringRuntimeError} from '../../../../Image'
+
+const closeSvg = stringToSvgStringRuntimeError(`
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M18.75 5.25L5.25 18.75" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (kaladivo) [inline apps/mobile/src/components/TosFlow/components/TosScreen/images/closeSvg.ts]: If you put `stroke="white"` away, you can specify it in Image component css definition instead

**Comment by prusnak** [comment]
It seems we have lots of stuff in`.yarn/cache`. Shouldn't we put this folder to `.gitignore`? (this is not just an issue with this PR, but about the whole repo in general)

File: apps/mobile/src/components/TosFlow/index.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import {createNativeStackNavigator} from '@react-navigation/native-stack'
+import TosScreen from './components/TosScreen'
+import {type TosStackParamsList} from '../../navigationTypes'
+import FaqScreen from './components/FqaScreen'
+
+const TosStack = createNativeStackNavigator<TosStackParamsList>()
+
+function TosFlow(): JSX.Element {
+  return (
+    <TosStack.Navigator
```
Comment (kaladivo) [inline apps/mobile/src/components/TosFlow/index.tsx]: Move to root navigator. Make sure screens can be accessed even when user is not logged in.

File: apps/mobile/src/navigationTypes.ts [hunk 6]
```diff
@@ -29,6 +29,8 @@ export type RootStackParamsList = {
   ChatDetail: {chatId: ChatId}
 
   // TODO terms and conditions etc
+
+  TosFlow: NavigatorScreenParams<TosStackParamsList>
```
Comment (kaladivo) [inline apps/mobile/src/navigationTypes.ts]: Should not be nested here

**Review by kaladivo — CHANGES_REQUESTED** [review]

**Comment by kaladivo** [comment]
@prusnak 

It's recommended by yarn documentation to keep them in: 
See: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

**Comment by prusnak** [comment]
> See: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Ok, thanks, TIL

**Review by kaladivo — APPROVED** [review]

**Pull request merged**